### ISSUE TITLE
Plugin: Allow command executions after starting/finishing a pomodoro

### ISSUE THREAD

**Issue by vsund** [eliostvs/tomate-gtk#18]
Hey, first of all, I _really_ like `tomate-gtk`. Imo it's the perfect mixture of simplicity and functionality for pomodoros.

But one thing I miss: When a pomodoro starts, I'd like to do some things (shut off all distractions etc) and when it finishes putting everything back to normal again. Since these things change often it doesn't make sense to do that as specific plugins, so a general plugin that allows executing some commands in a shell environment would solve all these things together.

Is such a plugin in scope of this project?

**Comment by eliostvs** [comment]
I just create the [plugin](https://github.com/eliostvs/tomate-exec-plugin). I expect to do the release this week.

**Comment by eliostvs** [comment]
I did the release yesterday. Tell me if you have any problems.

**Comment by vsund** [comment]
Yeah, installed it a few hours ago from the AUR. I wonder how I configure commands, and I can't find documentation on that :)

And besides, the link that points to the issues at the [exec repository](https://github.com/eliostvs/tomate-exec-plugin) points to the issues of the [alarm repository](https://github.com/eliostvs/tomate-alarm-plugin) ;)

**Comment by eliostvs** [comment]
Ow. I'll fix later today. In the Preferences > Extensions active the plugin. In the same window you'll see a new settings button that will be selectable when the plugin is selected. Click it and you'll the plugin settings window.

<img width="350" alt="screen shot 2017-05-09 at 16 29 14" src="https://cloud.githubusercontent.com/assets/1129420/25873655/fe8337a4-34d4-11e7-8c47-70428a3f39d1.png">
<img width="349" alt="screen shot 2017-05-09 at 16 30 01" src="https://cloud.githubusercontent.com/assets/1129420/25873656/fe86237e-34d4-11e7-9add-30d9ed01bc76.png">

**Comment by vsund** [comment]
Interesting, I have no such little button in the right-bottom corner.
(I'm on Archlinux with i3.)

**Comment by eliostvs** [comment]
I'm using Arch with Gnome/Mate and the button do show for me. Can you open the app from the command line with the paramenter -v? When you click in the plugin it should display a message `Activating settings for plugin Run Command` in the terminal.

**Comment by vsund** [comment]
When I open the preferences window I get
```
DEBUG:2017-05-10 11:15:35,474:tomate_gtk.dialogs.preference:plugin StatusIcon added
DEBUG:2017-05-10 11:15:35,475:tomate_gtk.dialogs.preference:plugin Run Command added
DEBUG:2017-05-10 11:15:35,475:tomate_gtk.dialogs.preference:plugin Alarm added
DEBUG:2017-05-10 11:15:35,475:tomate_gtk.dialogs.preference:plugin Notify added
```
but nothing more when clicking on an extension or navigating between the two tabs.

**Comment by eliostvs** [comment]
I installed i3 and it seems to be working.
![captura de tela de 2017-05-11 20-42-19](https://cloud.githubusercontent.com/assets/1129420/25978878/b079a0cc-368a-11e7-8333-e741b6356ff2.png)

**Comment by vsund** [comment]
I installed it on my laptop (also Archlinux+i3) and it worked there. Then I tried it on my desktop pc and I couldn't reprocude the error, I think it was something with an old version that got used (I updated but it opened the old version though).

Now it works :) Sorry for the inconveniences. And thanks again for creating that plugin!

---
And btw: The link that points to the issues at the [exec repository](https://github.com/eliostvs/tomate-exec-plugin) still points to the issues of the [alarm repository](https://github.com/eliostvs/tomate-alarm-plugin) ;)
